addons/hub/install/README.md

```diff
@@ -0,0 +1,42 @@
+# Argo Workflow Installation Add-on
+By applying this add-on to your OCM hub cluster, the Argo Workflow installation will automatically be applied to all your existing `managed` (`spoke`) clusters and all your to be registered `managed` (`spoke`) clusters.
+
+Under the `manifests` folder it contains the Argo Workflow install version of:
+
+```
+v3.4.2
+```
+
```

Comment:
Line 2 and 4: "Argo Workflow" -> "Argo Workflows"
Line 2: "to be registered" -> "to-be-registered"